temporalio/runtime.py

```diff
@@ -175,6 +175,8 @@ class TelemetryConfig:
     metrics: Optional[Union[OpenTelemetryConfig, PrometheusConfig]] = None
     """Metrics configuration."""
 
+    global_tags: Mapping[str, str] = field(default_factory=dict)
```

Comment:
Are these tags applied to metrics _and_ tracing? Does that include Prometheus metrics and OTel tracing?

Reply:
Oh, just realized. Can we at least add one line of docs here?

Reply:
Done